users_permissions/README.md

```diff
@@ -75,59 +75,45 @@ Then run this command:
 1. Create a new app following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
     For the purpose of this sample you don't need to create an app from a template. You can just create an empty application.
 1. Click the button `Review draft & deploy`.
-1. Go to the `Authentication` menu in the left panel and select :
+1. Go to the **Authentication** menu in the left panel and select :
     1. `Authentication Providers`. Then make sure the option "Email/Password" is ON. Save and then click the button `Review draft & deploy`. Read [this page](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) for more information about the other types of authentication.
     1. `Custom User Data`. Then make sure the option is enabled. Select cluster, database and collection where to store the custom data. Then select the field that to be used for mapping users with their custom data. For this sample the field name is `owner_id`. Read [this page](https://www.mongodb.com/docs/atlas/app-services/users/enable-custom-user-data/) for more information about enabling custom user data.
-1. Go to the `Sync` menu and [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/#enable-flexible-sync).
+1. Go to the **Rules** menu.
+    1. Select "Default roles and filters" under the service name. Choose from the list with `Other preset`: `readOwnWriteOwn (User can only read and write their own data)` and click the button `Add preset role`. If you switch to `Advanced view` the json should looks like the one in this file: /assets/atlas_app/data_sources/mongodb-atlas/default_rule.json.
+    1. To set specific permissions to a collection it must exist. Create a new collection with name "Item" if it doesn't exist. This could be done from the menu option `Create collection`, which is on the root level next to the service name. Then select the collection and set the rules. Choose from the list with `Other preset`: `Users can read and write their own data, admins can read and write all data)`.
```

Comment:
```suggestion
    1. To set specific permissions to a collection, it must exist. Create a new collection with name "Item" if it doesn't exist. This could be done from the menu option `Create collection`, which is on the root level next to the service name. Then select the collection and set the rules. Choose from the list with `Other preset`: `Users can read and write their own data, admins can read and write all data)`.
```

> `Users can read and write their own data, admins can read and write all data)`.
also something wrong with the parenthesis here.